src/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj

```diff
@@ -27,6 +27,25 @@
     <Optimize>true</Optimize>
     <ErrorReport>prompt</ErrorReport>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug_Future|AnyCPU'">
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <Optimize>false</Optimize>
+    <ErrorReport>prompt</ErrorReport>
+    <CheckForOverflowUnderflow>true</CheckForOverflowUnderflow>
+    <Future>true</Future>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release_Future|AnyCPU'">
+    <Future>true</Future>
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <Optimize>true</Optimize>
+    <ErrorReport>prompt</ErrorReport>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Future)' == 'true'">
```

Comment:
When doing the configuration work I was thinking we would end up with something like TargetVersion that would automatically fix the Paths and AssemblyVersion for these projects that want to build multiple versions at the same time. With the future approach it limits to just one future one but I can imagine cases were we might need to support multiple at the same time which using a TargetVersion approach would enable. @ericstj thoughts?

To be clear I'm fine with this approach for now but I think we need a more extensible mechanism for this in our build system.